keyboards/smart/info.json

```
@@ -0,0 +1,11 @@
+{
+    "manufacturer": "SmartCloud",
+    "url": "www.i-chip.cn",
+    "maintainer": "Jacky",
+    "bootloader": "custom",
+    "bootloader_instructions": "Press ESC(ROW0COL0), Plug USB. ",
+    "development_board": "bluepill",
```

You are not using a bluepill, so should not be using this preset
```suggestion
    "processor": "STM32F103",
    "board": "STM32_F103_STM32DUINO",
```